CodeStarterCamp_Week3/main.swift

```diff
@@ -8,58 +8,19 @@
 
 import Foundation
 
- 
-enum Coffee: String {
-    case americano = "아메리카노"
-    case cafeLatte = "카페라떼"
-    case cappuccino = "카푸치노"
-    case mocha = "모카"
-    case prapucihno = "프라푸치노"
-    case banillaLatte = "바닐라라떼"
-}
+var misterLee: Person = Person(money: 2000, name: "이철호")
+var missKim: Person = Person(money: 1500, name: "missKim")
+var Coda: Person = Person(money: 1500, name: "Coda")
+var yagombuks: CoffeeShop = CoffeeShop(sales: 0, barista: misterLee, menu: menu, pickUpTable: pickUpTable)
+
+missKim.order(_ : .americano, of: yagombuks, by: missKim.name)
+Coda.order(_ : .cappuccino, of: yagombuks, by: Coda.name)
+missKim.order(_ : .cafeLatte, of: yagombuks, by: missKim.name)
```

Comment:
order 메서드를 정의할 때, 파라미터에 wildcard를 사용하셨다면 앞을 아예 생략할 수 있습니다!
네이밍 잘해주셨네요 👍
```suggestion
missKim.order(.americano, of: yagombuks, by: missKim.name)
Coda.order(.cappuccino, of: yagombuks, by: Coda.name)
missKim.order(.cafeLatte, of: yagombuks, by: missKim.name)
```